OMDbSharp/Classes/Episode.cs

```diff
@@ -1,51 +1,29 @@
-﻿namespace OMDbSharp
+﻿namespace OMDbSharp.Classes
```

Comment:
I would like for the Namespace and Folder to be named "Objects" instead of "Classes" 😄

Reply:
No problem. I'll have the change committed once I'm back from work.